integration-tests/sceptre-project/templates/sam_template.yaml

```diff
@@ -5,5 +5,5 @@ Resources:
     Type: 'AWS::Serverless::Function'
     Properties:
       Handler: index.handler
-      Runtime: python3.6
+      Runtime: python3.9
```

Comment:
Apparently, the 3.6 runtime is no longer available for use. AWS rejects it, so I needed to update it.